_tests_/categoryClass.test.js

```diff
@@ -0,0 +1,17 @@
+const category = require('../src/components/categoryClass');
```

Comment:
- [ ] Please convert this to es6 import syntax. For this to work well you would need to use es6 export syntax for the class module. So you can simply convert `module.export = {Category}` to `export default Category`.